libs/cart/testing/src/in-memory-backend/cart/cart.service.ts

```diff
@@ -56,6 +56,7 @@ export class DaffInMemoryBackendCartService implements InMemoryDbService {
     const cart = this.getCart(reqInfo);
 
     cart.items = [];
+		cart.items = Object.assign([], cart.items);
```

Comment:
I don't think this one is necessary. The previous line already sets `cart.items` to a new object.

Reply:
removed